django_ipam/base/models.py

```diff
@@ -1,17 +1,24 @@
 import swapper
+from django.conf import settings
 from django.db import models
 from openwisp_utils.base import TimeStampedEditableModel
 
+from .fields import NetworkField
+
 
 class AbstractSubnet(TimeStampedEditableModel):
-    name = models.CharField(max_length=100, blank=True)  # Add a default value
+    subnet = NetworkField(default="10.0.0.0/24")
     description = models.CharField(max_length=100, blank=True)
+    master_subnet = models.ForeignKey('self', on_delete=models.CASCADE,
+                                      blank=True, null=True,
+                                      related_name="child_subnets")
+    ip_request = models.BooleanField(default=False)
```

Comment:
what's this for? Can we add this field only when we are going to actually use it for something? This kind of unused code clutters my mind.